packages/rest/src/lib/handlers/SequentialHandler.ts

```diff
@@ -402,6 +393,20 @@ export class SequentialHandler {
 				majorParameter: this.majorParameter,
 				global: isGlobal,
 			});
+			this.debug(
```

Comment:
Any reason this is called after `this.onRateLimit`? Ideally it would be before it, since that method may throw

Reply:
Not sure why the original did it that way, but it made sense to me because if the error is thrown, its thrown with  the exact same data (or at least mostly) and is thus no longer for debug purposes. The app has decided to throw on hitting said 429  so there is ~~hopefully~~ other handling for it.